src/components/FormTextField/FormTextField.js

```diff
@@ -11,6 +11,42 @@ export function FormTextField(props) {
         value: props.value,
       }
     : {};
+
+  // Define a map of width-to-classname mappings
+  const width = {
+    standard: "ds-w-full ds-max-w-[408px]",
+    12: "ds-w-[166px]",
+    11: "ds-w-[154px]",
+    8: "ds-w-[120px]",
+    7: "ds-w-[109px]",
+    4: "ds-w-[74px]",
+    3: "ds-w-[63px]",
+    2: "ds-w-[51px]",
+  };
+
+  // Use the width-to-classname map to get the classname based on the selected width
+  const classname = `${
+    width[props.width] || props.width
+  } ds-rounded ds-text-input ds-text-mobileh5 ds-text-multi-neutrals-grey100 ds-min-h-44px ds-text-form-input-gray ds-border-2 ds-py-5px ds-px-14px ${
+    props.hasError
+      ? "ds-border-specific-red-red50b"
+      : "ds-border-multi-neutrals-grey85a focus:ds-border-multi-blue-blue60f"
+  } ${props.exclude ? "exclude" : ""}`;
+
+  // Define a map of width-to-maxLength mappings
+  const maxLengths = {
+    12: 12,
+    11: 11,
+    8: 8,
```

Comment:
SINs have 9 digits. Might be good to have a 9-character option

Reply:
> SINs have 9 digits. Might be good to have a 9-character option

There is no 9 digits option in Figma. Asked Ravina, she says that Ontatio government uses 11 digit field for SIN in case user add spaces or dashes.